### DISCUSSION TITLE
Individual PK Values and Aggregated PK Values

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1481] [Questions & Problems]
What is the difference between the Individual PK Values and Aggregated PK Values, and which of the two is used to calculate the AUC fold change in the population?
![微信截图_20230320093829](https://user-images.githubusercontent.com/95362728/226227408-e3dd6d89-fe43-4476-a554-b655ef84b271.jpg)

**Comment by Yuri05** [comment]
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-parameters
![grafik](https://user-images.githubusercontent.com/25061876/226311310-ce5305d7-58f5-4169-ba79-11d058f4fd6e.png)
